src/index.ts

```diff
@@ -81,6 +81,18 @@ function setupOpenMRS() {
       }
     ],
     extensions: [
+      {
+        id: "test-results-widget",
+        slot: "test-results-widget",
```

Comment:
ID and Slot should be different.

The slot should be suffixed with `-slot`, or a given slot should be chosen. In this guess you want to have this part in the summary dashboard? Choose `patient-chart-summary-dashboard-slot` as slot.

Reply:
`patient-chart-summary-dashboard-slot` does not exist as a slot. All other widgets use a 1-to-1 mapping with their defined slots. Is this going to change in the near future?

Reply:
> patient-chart-summary-dashboard-slot does not exist as a slot.

If slots exist or don't exist does not make a difference. You also don't need to attach to a slot immediately.

> All other widgets use a 1-to-1 mapping with their defined slots.

This is / was just a temporary workaround to get them in as extensions.

> Is this going to change in the near future?

Yes - I think this was announced the last two weekly dev standups. We had the longer session on Monday (sorry during your vacation) where this system was proposed in greater depth. I think the video recording for it was shared already by Grace.